### PR TITLE
RAC-5032 - Converted the workflows CIT test scripts to FIT format for staging

### PR DESCRIPTION
**Pull request by hohene**
Converted the workflows CIT test scripts to FIT format.
This is first part of committing these changed scripts to show the diffs.

### CODE REVIEW

**Comment by hohene** [comment]
@larry-dean @keedya @jimturnquist @johren @derrickostertag

File: test/tests/api/fit-staging/v2_0/workflowTasks_tests.py [hunk 1]
```diff
@@ -1,78 +1,81 @@
...
+        self.workflowTaskDict = {
             "friendlyName": "fn_1",
             "injectableName": "in_1",
             "implementsTask": "im_1",
             "options": {},
             "properties": {}
         }
 
-    @test(groups=['workflowTasks_api2.tests', 'api2_workflowTasks_get'])
+    # @test(groups=['workflowTasks_api2.tests', 'api2_workflowTasks_get'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflowTasks_tests.py]: remove (per Jeanne)

File: test/tests/api/fit-staging/v2_0/workflowTasks_tests.py [hunk 2]
```diff
@@ -1,78 +1,81 @@
...
-        """ Testing GET:/worflows/tasks"""
+        # """ Testing GET:/worflows/tasks"""
         WorkflowApi().workflows_get_all_tasks()
-        assert_equal(200,self.__client.last_response.status)
-        assert_not_equal(0, len(json.loads(self.__client.last_response.data)), message='Workflow tasks list was empty!')
+        self.assertEqual(200, self.__client.last_response.status)
+        self.assertNotEqual(0, len(loads(self.__client.last_response.data)), msg='Workflow tasks list was empty!')
 
-    @test(groups=['workflowTasks_library_put'], depends_on_groups=['workflowTasks_library_get'])
+    # @test(groups=['workflowTasks_library_put'], depends_on_groups=['workflowTasks_library_get'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflowTasks_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
-        self.__client = config.api_client
-        self.__task_worker = None
-        self.__graph_name = None
-        self.__tasks = []
-        self.__graph_status = []
-        self.workflowDict = {
+HTTP_NOT_FOUND = 404
+
+
+# @test(groups=['workflows_api2.tests'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 4]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
             ]
         }
 
-    @test(groups=['delete_all_active_workflows_api2'])
-    def delete_all_active_workflows(self):
-        """Testing node PUT:/nodes/identifier/workflows/action"""
+    # @test(groups=['delete_all_active_workflows_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 5]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
...
-                assert_not_equal(id, None)
+                self.assertNotEqual(id, None)
                 try:
                     Api().nodes_workflow_action_by_id(id, {'command': 'cancel'})
                 except ApiException as err:
-                    LOG.warning(err)
+                    logs.warning(" Error: %s", err)
 
-    @test(groups=['workflows_get_api2'], depends_on_groups=['delete_all_active_workflows_api2'])
+    # @test(groups=['workflows_get_api2'], depends_on_groups=['delete_all_active_workflows_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 6]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
...
+        self.assertEqual(201, self.__client.last_response.status)
+        rawj = loads(self.__client.last_response.data)
         instance_id = rawj.get('instanceId')
-        assert_is_not_none(instance_id)
-        assert_equal('Graph.noop-example', str(rawj['definition'].get('injectableName')))
+        self.assertIsNotNone(instance_id)
+        self.assertEqual('Graph.noop-example', str(rawj['definition'].get('injectableName')))
 
-    @test(groups=['workflows_get_id_api2'], depends_on_groups=['workflows_get_api2'])
+    # @test(groups=['workflows_get_id_api2'], depends_on_groups=['workflows_get_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 7]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
...
+        rawj = loads(self.__client.last_response.data)
         instance_id = rawj[0].get('instanceId')
-        assert_is_not_none(instance_id)
+        self.assertIsNotNone(instance_id)
         Api().workflows_get_by_instance_id(instance_id)
-        assert_equal(200,self.__client.last_response.status)
+        self.assertEqual(200, self.__client.last_response.status)
 
-    @test(groups=['workflows_get_id_api2'],depends_on_groups=['workflows_get_api2'])
+    # @test(groups=['workflows_get_id_api2'],depends_on_groups=['workflows_get_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 8]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
...
-            assert_equal(404, self.__client.last_response.status, message='status should be 404. No exception raised')
+            self.assertEqual(404, self.__client.last_response.status, msg='status should be 404. No exception raised')
         except ApiException as e:
-            assert_equal(404,e.status, message = 'status should be 404')
+            self.assertEqual(404, e.status, msg='Expected 404 status, received {}'.format(e.status))
         except (TypeError, ValueError) as e:
             assert(e.message)
 
-    @test(groups=['workflows_graphs_get_api2'])
+    # @test(groups=['workflows_graphs_get_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 9]
```diff
@@ -49,76 +53,83 @@ def __init__(self):
...
-        assert_not_equal(0, len(json.loads(self.__client.last_response.data)),
-                         message='Workflows list was empty!')
+        self.assertEqual(200, self.__client.last_response.status)
+        resp = loads(self.__client.last_response.data)
+        logs.debug_6(" Workflow graphs: %s", dumps(resp, indent=4))
+        self.assertNotEqual(0, len(loads(self.__client.last_response.data)),
+                            msg='Workflows list was empty!')
 
-    @test(groups=['workflows_graphs_put_api2'])
+    # @test(groups=['workflows_graphs_put_api2'])
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 10]
```diff
@@ -128,102 +139,112 @@ def test_workflows_graphs_put(self):
...
-                assert_equal(readInjectableName,self.workflowDict.get('injectableName'))
+                self.assertEqual(readFriendlyName, self.workflowDict.get('friendlyName'))
+                self.assertEqual(readInjectableName, self.workflowDict.get('injectableName'))
 
-        assert_equal(foundInsertedWorkflow, True)
+        self.assertEqual(foundInsertedWorkflow, True)
 
-    @test(groups=['workflows_graphs_get_by_name_api2'],
-          depends_on_groups=['workflows_graphs_put_api2'])
+    # @test(groups=['workflows_graphs_get_by_name_api2'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 11]
```diff
@@ -128,102 +139,112 @@ def test_workflows_graphs_put(self):
...
-        assert_equal(200,self.__client.last_response.status)
-        rawj = json.loads(self.__client.last_response.data)
-        assert_equal(self.workflowDict.get('friendlyName'), str(rawj[0].get('friendlyName')))
+        self.assertEqual(200, self.__client.last_response.status)
+        rawj = loads(self.__client.last_response.data)
+        self.assertEqual(self.workflowDict.get('friendlyName'), str(rawj[0].get('friendlyName')))
 
-    @test(groups=['workflows_graphs_put_by_name_api2'],
-          depends_on_groups=['workflows_graphs_get_by_name_api2'])
+    # @test(groups=['workflows_graphs_put_by_name_api2'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 12]
```diff
@@ -128,102 +139,112 @@ def test_workflows_graphs_put(self):
...
+        self.assertEqual(201, self.__client.last_response.status)
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        rawj = json.loads(self.__client.last_response.data)
-        assert_equal(self.workflowDict2.get('friendlyName'), str(rawj[0].get('friendlyName')))
+        rawj = loads(self.__client.last_response.data)
+        self.assertEqual(self.workflowDict2.get('friendlyName'), str(rawj[0].get('friendlyName')))
 
-    @test(groups=['workflows_graphs_delete_by_name_api2'],
-          depends_on_groups=['workflows_graphs_put_by_name_api2'])
+    # @test(groups=['workflows_graphs_delete_by_name_api2'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/workflows_tests.py [hunk 13]
```diff
@@ -250,27 +271,28 @@ def handle_graph_finish(self,body,message):
...
-        if 'failed' in self.__graph_status:
-            fail('Failure running {0}'.format(self.__graph_name))
+                logs.error('Timeout for %s, node %s', self.__class__.__graph_name, task.id)
+                self.__class__.__graph_status.append('failed')
+        if 'failed' in self.__class__.__graph_status:
+            self.fail('Failure running {}'.format(self.__class__.__graph_name))
 
-    @test(groups=['node_workflows_post_api2'],
-            depends_on_groups=['workflows_graphs_put_api2', 'delete_all_active_workflows_api2'])
+    # @test(groups=['node_workflows_post_api2'],
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/workflows_tests.py]: remove

**Comment by johren** [comment]
test this please

**Comment by johren** [comment]
test this please

**Comment by johren** [comment]
test this please

**Pull request merged**